### ISSUE TITLE
You should add a colored sample to the palettes

### ISSUE THREAD

**Issue by LukyVj** [Skoks/sassy-palette#1]
Hey, awesome project on here ! Really loving the idea. 
I have a suggestion tho, you should adds a colored square corresponding to the colors in the maps. 

You should follow the link : 
http://codepen.io/LukyVj/blog/how-to-add-colored-images-to-sassdoc

( and there is a pure "sassdoc" solution for it )

**Comment by Skoks** [comment]
Hey, thanks for your feedback! 

[Sassdoc](http://sassdoc.com/) has already implemented this feature. 

```
///
/// Color Sample
///
/// @type {Color}
///

$color: #BADA55;
```
![Color Sample](https://api.monosnap.com/rpc/file/download?id=5nricuiZgPrGMbxBUv2O9n7f11j3FK)

In my case It will be more useful to output color samples in examples section. Please let me know if there is a solution for that. Thank you.

**Comment by LukyVj** [comment]
Yes I discovered it after doing my own version of the colored rectangles ( see the edit on the article ) 

You can create a "color section" then you "clone" your colors with javacript, and you put them in the section, let me know if you need a demo or so :)

**Comment by Skoks** [comment]
PR with example will be awesome :octocat:
